src/main/java/uk/gov/companieshouse/web/pps/controller/pps/ConfirmationController.java

```diff
@@ -104,7 +104,7 @@ public String getConfirmation(@PathVariable String companyNumber,
         }
 
         model.addAttribute(COMPANY_NUMBER_ATTR, companyNumber);
-        model.addAttribute(PENALTY_NUMBER_ATTR, penaltyNumber);
+        model.addAttribute(PENALTY_NUMBER_ATTR, penaltyId);
```

Comment:
Ok, so we're leaving this model attribute as a hangover of what it was?
Hopefully we remember when we next look :)